spec/hand_rankings/hand_entities/high_card_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'poker_hands/hand_rankings/hand_entities/high_card'
+require 'poker_hands/card'
+require 'spec_helper'
+
+
+RSpec.describe PokerHands::Entities::HighCard do
+  subject { PokerHands::Entities::HighCard.new(cards: hand) }
+  let(:hand) do
+    [
+      PokerHands::Card.new(4, 'H'),
+      PokerHands::Card.new(11, 'S'),
+      PokerHands::Card.new(8, 'D'),
+      PokerHands::Card.new(2, 'S'),
+      PokerHands::Card.new(9, 'C')
+    ]
+  end
+
+  describe 'HighCard' do
+    it 'receives a poker hand' do
+      expect(PokerHands::Entities::HighCard).to receive(:new).with(cards: hand)
+      PokerHands::Entities::HighCard.new(cards: hand)
```

Comment:
We can probably drop this test, as our other test will pick up on our initialize method failing.

This test in itself doesn't provide much value either, as we aren't testing how our code words; we're testing that our test works. I can understand the sentiment; we want to ensure that our initialize works. But this test won't actually do that, as when we use the keyword `expect`, we're actually stubbing out the underlying call. So this test is more or less testing that message passing in Ruby works, and our code doesn't get hit (feel free to throw a `binding.pry` inside `initialize` to verify this).